website/docs/components/page-header/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,236 @@
+
+We recommend limiting the Page Header description to 1-2 sentences. If a longer description is required, consider linking to external documentation or moving more detailed content to an interstitial component like a [Flyout](/components/flyout).
+
+![Page Header Description](/assets/components/page-header/page-header-description.png =700x*)
+
+!!! Dont
+
+Don't include overly complex details, long-form content, or instructions in the description. This can add too much visual weight to the Page Header and can detract from the content on the main page.
+
+![Page Header Long Form Description](/assets/components/page-header/page-header-description-dont-long-form.png =700x*)
+
+!!!
+
+!!! Do
+
+Instead, link to more complex content using an [Inline Link](/components/link/inline) with an optional icon to indicate the scope of the link (internal resource, external resource, etc).
+
+<Hds::PageHeader as |PH|>
+<PH.Title>HCP Packer Dashboard</PH.Title>
+<PH.IconTile @icon="packer-color" @color="packer" />
+<PH.Description>
+Channel created and managed automatically for you by Packer. For more information on how this channel is managed refer to the Packer <Hds::Link::Inline @icon="external-link" @href="#">documentation</Hds::Link::Inline>.
+</PH.Description>
+</Hds::PageHeader>
+
+!!!
+
+!!! Insight
+
+**Hand-off note:** Nesting a [Link Inline](/components/link/inline) with an icon inside of the description is not supported in the Figma component. Instead, override the description text with the correct text style and color, and ensure that this detail is accounted for in the engineering hand-off.
+
+!!!
+
+## Custom metadata
+
+If necessary to include metadata like key/value pairs, multiple page-level statuses, or other structured content, custom metadata can be passed to the component via the `customMetadata` property in Figma.
+
+![Page Header custom metadata](/assets/components/page-header/page-header-custom-metadata-example.png =700x*)
+
+### Key/value pairs as custom metadata
+
+Representing metadata with a set of key/value pairs is common in HashiCorp products and can be useful when communicating relational information between products, versioning, and other structured data.
+
+![Custom metadata key/value pairs](/assets/components/page-header/page-header-custom-metadata.png =600x*)
+
+!!! Dont
+
+We recommend not exceeding more than four key/value pairs; this can result in unnecessarily complex content within the Page Header. Instead, move this content to the page.
+
+![Complex metadata dont](/assets/components/page-header/page-header-custom-metadata-dont.png)
+
+!!!
+
+## Actions
+
+Use [Buttons](/components/button) and [Dropdowns](/components/dropdown) in the Page Header to highlight functions and actions that impact the page as a whole. Examples of this include:
+
+- Creating new objects that are listed within the page
+- Surfacing management details like connecting to an API
+- Pausing or refreshing a service pertaining to the page
```

Comment:
Should probably add periods after these list items for consistency with your other list items elsewhere